frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -224,6 +222,29 @@ function ReaderBookmark:addToMainMenu(menu_items)
                 end,
                 separator = true,
             },
+            {
+                text = _("Show highlight colors"),
+                checked_func = function()
+                    return G_reader_settings:isTrue("bookmarks_items_show_color")
+                end,
+                callback = function()
+                    G_reader_settings:flipNilOrFalse("bookmarks_items_show_color")
+                end,
+            },
+            {
+                text = _("Include default highlight color"),
```

Comment:
To my non-native English ears & eyes, the use of "Include" here feels odd.
No obvious alternative suggestion :/

In the past, I've used "Also..." (which feels odd re-reading these :)) when I had no better idea (at least one of these must be mine):
```
frontend/apps/filemanager/filemanagerfilesearcher.lua:            text = _("Also search in book metadata"),
frontend/apps/filemanager/filemanagercollection.lua:        text = _("Also search in book content (slow)"),
frontend/ui/elements/common_settings_menu_table.lua:            text = _("Also show values in pixels"),
plugins/statistics.koplugin/main.lua:                        text = _("Also use in calendar view"),
```
so I would have used "[ ] Also show default highlight color".